pkgs/top-level/lua-packages.nix

```diff
@@ -229,6 +229,12 @@ let
     };
     buildInputs = [ unzip ];
 
+    preBuild = ''
+      makeFlagsArray=(CC=$CC);
+    '';
+
+    buildFlags = if stdenv.isDarwin then "macosx" else "";
```

Comment:
You've used this in a couple of lua packages already. Is it just something that should be shared across all lua packages, or is it specific to couple you've made PRs for?